### PR TITLE
trainings auth

### PR DESCRIPTION
**Pull request by Firuz2006**

### CODE REVIEW

File: src/MRA.Jobs.Api/WebUI/Controllers/ApplicationsController.cs [hunk 1]
```diff
@@ -19,19 +20,22 @@ public ApplicationsController(ILogger<ApplicationsController> logger)
     }
 
     [HttpGet]
+    [AllowAnonymous]
```
Comment (komdil) [inline src/MRA.Jobs.Api/WebUI/Controllers/ApplicationsController.cs:23]: [Authorize(ApplicationPolicies.Reviewer)]
Only reviewer can see applications

File: src/MRA.Jobs.Api/WebUI/Controllers/ApplicationsController.cs [hunk 2]
```diff
@@ -19,19 +20,22 @@ public ApplicationsController(ILogger<ApplicationsController> logger)
...
     [HttpGet]
+    [AllowAnonymous]
     public async Task<ActionResult<PagedList<ApplicationListDto>>> GetAll([FromQuery] PagedListQuery<ApplicationListDto> query)
     {
         var applications = await Mediator.Send(query);
         return Ok(applications);
     }
 
     [HttpGet("{slug}")]
+    [AllowAnonymous]
```
Comment (komdil) [inline src/MRA.Jobs.Api/WebUI/Controllers/ApplicationsController.cs:31]: reviewer only

File: src/MRA.Jobs.Api/WebUI/Controllers/ApplicationsController.cs [hunk 3]
```diff
@@ -19,19 +20,22 @@ public ApplicationsController(ILogger<ApplicationsController> logger)
...
     public async Task<ActionResult<ApplicationDetailsDto>> Get(string slug, CancellationToken cancellationToken)
     {
         return await Mediator.Send(new GetBySlugApplicationQuery { Slug = slug }, cancellationToken);
     }
 
     [HttpPost]
+    [Authorize]
```
Comment (komdil) [inline src/MRA.Jobs.Api/WebUI/Controllers/ApplicationsController.cs:38]: This one is correct

File: src/MRA.Jobs.Api/WebUI/Controllers/ApplicationsController.cs [hunk 4]
```diff
@@ -72,6 +76,7 @@ public async Task<ActionResult<bool>> AddNote(string slug, AddNoteToApplicationC
     }
 
     [HttpGet("{status}")]
+    [AllowAnonymous]
```
Comment (komdil) [inline src/MRA.Jobs.Api/WebUI/Controllers/ApplicationsController.cs:79]: reviewer only

File: src/MRA.Jobs.Api/WebUI/Controllers/InternshipsController.cs [hunk 5]
```diff
@@ -74,6 +79,7 @@ public async Task<ActionResult<TestInfoDto>> SendTestCreationRequest([FromRoute]
     }
 
     [HttpPost("{slug}/test/result")]
+    [AllowAnonymous]
     public async Task<ActionResult<TestResultDto>> GetTestResultRequest([FromRoute] string slug, [FromBody] CreateTestResultCommand request, CancellationToken cancellationToken)
     {
```
Comment (komdil) [inline src/MRA.Jobs.Api/WebUI/Controllers/InternshipsController.cs:84]: remove this route

**Review by komdil — CHANGES_REQUESTED** [review]

**Review by komdil — APPROVED** [review]

**Review by adheeeem — APPROVED** [review]

**Pull request merged**